### DISCUSSION TITLE
change UserImage

### DISCUSSION THREAD

**Discussion by hosnabigdeli** [serenity-is/Serenity#6313] [Help]
![88](https://user-images.githubusercontent.com/56981629/157212165-0b055e8f-1954-4858-a66b-7c5a591f5777.PNG)

hi
I want when a user clicks on the edit icon to be able to upload a photo and save this photo for this user and change the avatar image

This is my code that I wrote with jquery that sends the photo that the user uploads to the endpoint, but I dont know how I can save this photo in userRow . Anyone have an idea?

_Layout.cshtml:
`
            $('#edit').change(function (e) {
                debugger;
                var file_id = e.target.id;
                    var form_data = new FormData();
                    var file_data = $("#" + file_id).prop("files")[0];
                form_data.append("file_name", file_data);
                console.log(file_data);
                debugger;

                        $.ajax({
                           // url: '/Account/CheckOTP', // point to server-side controller method

                            url: '/Services/Administration/SimpleUser/upload_image', // point to server-side controller method
                            dataType: 'text', // what to expect back from the server
                            cache: false,
                            contentType: false,
                            processData: false,
                            data: form_data,
                            type: 'post',
                            success: function (response) {
                                $('#msg').html(response); // display success response from the server
                            },
                            error: function (response) {
                                $('#msg').html(response); // display error response from the server
                            }

                        });

            });`

**Comment by dfaruque** [comment]
in your service "/Services/Administration/SimpleUser/upload_image". you may write something like the following

            connection.UpdateById(new UserRow
            {
                UserId = int.Parse(User.GetIdentifier()),
                UserImage = uploadedImagePath
            });
